packages/modules/create-itwin-viewer/package-lock.json

```diff
@@ -0,0 +1,310 @@
+{
+  "name": "test",
```

Comment:
revert these changes. this is a rush repo so it should not have a package-lock

Reply:
package-lock.json removed